variants/devise/spec/models/user_spec.rb

```diff
@@ -0,0 +1,4 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
```

Comment:
Since we make a factory for this, shall we add a simple `it { is_expected.to be_valid }` here?